frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+
+export interface MultiColumnFieldRowProps {
+  name: string;
+  rowIndex: number;
+  children: React.ReactNode;
+  onDelete: () => void;
+}
+
+const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
+  name,
+  rowIndex,
+  onDelete,
+  children,
+}) => (
+  <div className="odc-multi-column-field__row">
+    {React.Children.map(children, (child: React.ReactElement) => {
+      const fieldName = `${name}.${rowIndex}.${child.props.name}`;
+      const newProps = { ...child.props, name: fieldName };
+      return (
+        <div key={fieldName} className="odc-multi-column-field__col">
+          {React.cloneElement(child, newProps)}
+        </div>
+      );
+    })}
+    <div className="odc-multi-column-field__col--button">
+      <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
```

Comment:
This ideally should be a button and not just a clickable image.

Reply:
Making it a button adds extra styling specific to buttons and we need to reset that.

Reply:
PF4 button with `variant="plain"` works here except it adds padding.
I'm surprised that I cannot create an inline icon button with pf4 😱 Seems to only allow link inline buttons which makes the icon blue.

Ok for now we can do this to align with the existing component. Please create a technical debt issue to address this in both locations as I do not want the two components to look different right now. Ideally we also that component on this one.